lighthouse-core/report/html/report-styles.css

```diff
@@ -351,12 +351,7 @@
   word-break: break-word;
 }
 
-.lh-audit-group a,
-.lh-category-header__description a,
-.lh-audit__description a,
-.lh-warnings a,
-.lh-footer a,
-.lh-table-column--link a {
+a {
```

Comment:
since we embed lighthouse this needs to be `.lh-root a`

Reply:
Getting some weird effects with that change. Perhaps this is why links weren't colored this way to begin with?

![Screen Shot 2021-03-29 at 1 44 02 PM](https://user-images.githubusercontent.com/6752989/112877502-d5b68400-9094-11eb-8004-1ed7fde056c4.png)

Reply:
CSS specificity is the cause

![image](https://user-images.githubusercontent.com/4071474/112890071-077f1900-909c-11eb-8afd-9281118ac8a9.png)

try setting the color for the percentage more explicitly:

```
.lh-gauge__percentage div { color: var(--color-average) }
```

Reply:
You are probably right, but this is starting to become a non-trivial solution to a problem that appears to already be fixed. Considering the gauges changed, it seems more likely that we create new problems instead of preventing future link issues.

In other words: *"If it aint broke, don't fix it"*

How do you feel about closing this?